Dockerfile

```diff
@@ -40,21 +38,13 @@ LABEL maintainer.0="Dendi Suhubdy (dendi@bitwyre.com)" \
     maintainer.1="Yefta Sutanto (yefta@bitwyre.com)" \
     maintainer.2="Aditya Kresna (kresna@bitwyre.com)"
 
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends gosu && \
-    rm -rf /var/lib/apt/lists/* && \
-    useradd -r bitcoin
-RUN mkdir -p /home/bitcoin/.bitcoin && \
-    chown -R bitcoin /home/bitcoin
+RUN mkdir -p /etc/bitcoind
 
 COPY --from=builder /app/src/bitcoin/depends/x86_64-pc-linux-gnu /usr/local
-COPY docker-entrypoint.sh /docker-entrypoint.sh
-RUN chmod +x /docker-entrypoint.sh
 
-VOLUME ["/home/bitcoin/.bitcoin"]
+VOLUME ["/root/.bitcoin"]
 EXPOSE 8332 8333 18332 18333 18443 18444 28332 28333 28334 28335
 
-ENV BITCOIN_DATA = "/home/bitcoin/.bitcoin"
+ENV BITCOIN_DATA = "/root/.bitcoin"
```

Comment:
wait why?

Reply:
Because of this
![image](https://user-images.githubusercontent.com/26949832/75239751-d297f700-57f5-11ea-8d29-33ff8fb810aa.png)